integrations/dynatrace/client.py

```diff
@@ -0,0 +1,118 @@
+import typing
+from enum import StrEnum
+from typing import Any, AsyncGenerator
+
+import httpx
+from loguru import logger
+from port_ocean.context.event import event
+from port_ocean.utils import http_async_client
+
+from integration import DynatraceResourceConfig
+
+
+class ResourceKey(StrEnum):
+    PROBLEM = "problem"
+    SLO = "slo"
+    ENTITY_TYPE = "entity_type"
```

Comment:
Since we're relying on the selector for the entity types, I guess we don't need this then